src/tools/simple-kv-verify/CMakeLists.txt

```diff
@@ -45,11 +45,11 @@ nebula_add_executable(
         gtest
 )
 
-install(
-    TARGETS
-        simple_kv_verify
-    DESTINATION
-        bin
-    COMPONENT
-        tool
-)
+#install(
+#    TARGETS
+#        simple_kv_verify
+#    DESTINATION
+#        bin
+#    COMPONENT
+#        tool
+#)
```

Comment:
These tools can be compiled？
Just don’t include them when packing.

Reply:
Yeah, they will be compiled but not packaged.